src/content/docs/new-relic-solutions/get-started/implementation-guide-alerting-proactive-solutions.mdx

```diff
@@ -0,0 +1,100 @@
+---
+title: "Implementation part 4: Alerting and other proactive solutions"
+tags:
+  - New Relic solutions
+  - Best practices guides
+metaDescription: "Part 4 of the New Relic implementation guide, where you think about your alerting strategies, and set up alerting, synthetic monitors, and errors inbox."
+---
+
+import alertingUI from 'images/alerting-ui.png'
+
+import monitorIndex from 'images/monitor-index.png'
+
+import errorsUi from 'images/errors-ui.png'
+
+*This is the fourth and final part of [our implementation guide](/docs/new-relic-solutions/get-started/implementation-guide-intro).*
+
+In previous implementation stages, you [instrumented your stack](/docs/new-relic-solutions/get-started/implementation-guide-instrument) and [became acquainted with the New Relic platform](/docs/new-relic-solutions/get-started/implementation-guide-organize-data). Now is a good time to think about proactive solutions that will notify you of problems early and help you avoid worst-case scenarios. In this stage, you'll learn about some important solutions in this area, including: 
+
+* Alerting
+* Synthetic monitors
+* Errors inbox
+
+## Think about your alerting strategy [#alert-strategy]
+
+<img
+  title="Alerts UI"
+  alt="Alerts UI"
+  src={alertingUI}
+/>
+
+<figcaption>
+The New Relic alerts UI gives you a view of the state of alert conditions in an account.
+</figcaption>
+
+Before setting up alerts, we recommend spending some time thinking about your alerting goals and strategy. The larger your organization is, the more important that is.
+
+When you don't have an alerting strategy, and instead set up alerts quickly and haphazardly to solve one-off problems, that can result in too many alert notifications going out. When that happens, your team will suffer from alert fatigue and start to ignore alerts. Spending some time thinking about your alerting strategy will ensure you set up alerts in a smart way that can scale as your organization grows or as you add more data to New Relic. 
```

Comment:
Is it worth mentioning some things that people should take into account when designing their alert strategy? Like key questions?